x-pack/test/functional/apps/ml/anomaly_detection_result_views/anomaly_explorer.ts

```diff
@@ -151,6 +151,26 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           await ml.anomaliesTable.assertTableNotEmpty();
         });
 
+        it('should allow filtering by influencer', async () => {
+          const fieldName = testData.expected.influencers[0].field;
+          const fieldValue = testData.expected.influencers[0].labelsContained[0];
+
+          await ml.testExecution.logTestStep(
+            'adds influencer filter by clicking on the influencer add filter button'
+          );
+          await ml.anomalyExplorer.addFilterForInfluencer(fieldName, fieldValue);
+          await ml.testExecution.logTestStep('query bar and table rows reflect filter');
+          await ml.anomalyExplorer.assertQueryBarContent(`${fieldName}:"${fieldValue}"`);
+          await ml.anomaliesTable.assertInfluencersCellsContainFilter(
+            `${fieldName}: ${fieldValue}`
+          );
+          await ml.testExecution.logTestStep(
+            'removes influencer filter by clicking on the influencer remove filter button'
+          );
+          await ml.anomalyExplorer.removeFilterForInfluencer(fieldName, fieldValue);
+          await ml.anomalyExplorer.assertQueryBarContent('');
```

Comment:
And probably worth checking that the Top influencer list and 'view by' swim lane axis labels have reverted to the expected values when the filter is removed.

Reply:
Added in 2f7485f24952fa0b40229d8a7cf42d0f2a4a4661